ddtrace/contrib/grpc/constants.py

```diff
@@ -12,7 +12,6 @@
 GRPC_REQUEST_METADATA_PREFIX_KEY = "grpc.request.metadata."
```

Comment:
Everything looks good to me. @brettlangdon Do we need a changelog for this if we're switching the constant value?

Reply:
Removing this constant is a breaking change. Instead we should deprecate it and add a release note (we should set the removal version to v2.0.0)

Reply:
Sure! Is this just about setting a release note, or is there any source to change (besides reestablishing that constant in this PR)?

Reply:
We'll need a releasenote and a code change. Here an example of how we deprecated constants in the past: https://github.com/DataDog/dd-trace-py/commit/be529ca80a2f1d32ec94e3f3a53e0e9830323f59. You'll need a code change like this:

```
from ddtrace.vendor.debtcollector import deprecate

def __getattr__(name):
    if name == "GRPC_PORT_KEY":
       deprecate(
            ("%s.%s is deprecated" % (__name__, name)),
            removal_version="2.0.0",
        )
        return "grpc.port"

    if name in globals():
        return globals()[name]

    raise AttributeError("%s has no attribute %s", __name__, name)
```

Reply:
Also can we add a test case for this deprecation warning. Here's an example of how we did this in the past: https://github.com/DataDog/dd-trace-py/blob/0.61/tests/tracer/test_constants.py#L20